docker-entrypoint.sh

```diff
@@ -3,7 +3,15 @@ set -e
 export PHP_HOST=${PHP_HOST:-php}
 export PHP_PORT=${PHP_PORT:-9000}
 export NGINX_PHP_READ_TIMEOUT=${NGINX_PHP_READ_TIMEOUT:-900}
-export NGINX_DEFAULT_SERVER_NAME=${NGINX_DEFAULT_SERVER_NAME:-drupal}
+# As default we have an "_" as server name.
+export NGINX_DEFAULT_SERVER_NAME=${NGINX_DEFAULT_SERVER_NAME:-_}
+# If you want to declare the default.conf server as the default
+# catch-all server, set the env var DECLARE_DEFAULT_SERVER to 1.
+if [ $DECLARE_DEFAULT_SERVER == 1 ]; then
+  export DEFAULT_SERVER="default_server"
+else
+  export DEFAULT_SERVER=""
+fi
```

Comment:
we could infere the `DEFAULT_SERVER` by checking if `NGINX_DEFAULT_SERVER_NAME == '_'`

Reply:
```
if [ $NGINX_DEFAULT_SERVER_NAME == '_']; then
  export DEFAULT_SERVER="default_server"
else
  export DEFAULT_SERVER=""
fi
```